### PR TITLE
fix(cmd): use list instead of find in cli

### PR DESCRIPTION
**Pull request by jademcgough**
long-running ask from @sanderson :)

Change all places in the CLI where we have `find` to use `list`. We inconsistently use the two in our CLI, so this adds more consistency in the UX.

Also adds `find` (to make this non-breaking) and `ls` (to be more unixy) as aliases for the command.

### CODE REVIEW

**Comment by jademcgough** [comment]
@rawkode good call

**Comment by jsteenb2** [comment]
we're going to want to update the tests as well to verify it hits the list endpoint, we may also want to run these same tests for the `find` cmd to maintain that backweards compatibility in our testing. Example here:

https://github.com/influxdata/influxdb/blob/master/cmd/influx/organization_test.go#L78

File: cmd/influx/bucket_test.go [hunk 1]
```diff
@@ -289,7 +304,11 @@ func TestCmdBucket(t *testing.T) {
 				cmdFn, calls := cmdFn()
 				cmd := builder.cmd(cmdFn)
 
-				cmd.SetArgs(append([]string{"bucket", "find"}, tt.flags...))
+				if tt.command == "" {
+					tt.command = "find"
```
Comment (jsteenb2) [inline cmd/influx/bucket_test.go]: should this be `list` as the default?

Comment (jademcgough) [reply cmd/influx/bucket_test.go]: yes, not sure what I was thinking there

File: cmd/influx/organization_test.go [hunk 2]
```diff
@@ -411,6 +431,8 @@ func TestCmdOrg(t *testing.T) {
 			}
 
 			testMemberFn(t, "list", cmdFn, tests...)
+			testMemberFn(t, "ls", cmdFn, tests[:0]...)
```
Comment (jsteenb2) [inline cmd/influx/organization_test.go]: I don't think this is testing anything here :thinking: , did you mean to put `testMemberFn(t, "ls", tests...)`?

`tests[:0] == empty slice`

File: cmd/influx/secret_test.go [hunk 3]
```diff
@@ -85,7 +101,12 @@ func TestCmdSecret(t *testing.T) {
 				)
 				nestedCmdFn, calls := cmdFn()
 				cmd := builder.cmd(nestedCmdFn)
-				cmd.SetArgs(append([]string{"secret", "find"}, tt.flags...))
+
+				if tt.command == "" {
+					tt.command = "find"
```
Comment (jsteenb2) [inline cmd/influx/secret_test.go]: default to `list` here?

File: cmd/influx/user_test.go [hunk 4]
```diff
@@ -269,7 +290,12 @@ func TestCmdUser(t *testing.T) {
 				)
 				nestedCmdFn, calls := cmdFn()
 				cmd := builder.cmd(nestedCmdFn)
-				cmd.SetArgs(append([]string{"user", "find"}, tt.flags...))
+
+				if tt.command == "" {
+					tt.command = "find"
```
Comment (jsteenb2) [inline cmd/influx/user_test.go]: same comment about defaults

File: cmd/influx/organization_test.go (hunk 2 above)
Comment (jademcgough) [reply cmd/influx/organization_test.go]: I wanted the first test, since running it against every test seems unnecessary. Fixing.

**Pull request merged**